src/Resources/assets/js/onboardingWizard/config/steps.js

```diff
@@ -103,12 +103,34 @@ export const steps = [
 		},
 		btnNextClass: 'shepherd-button--arrow-down',
 	},
+	{
+		showOn: function () {
+			currentStepValidator('.js-two-fields-test .required.field', '.pushable');
+			return paymentTypeIndicator(
+				'.js-onboardingWizard-environment',
+				'0'
```

Comment:
what does '0' and '1' do?

Reply:
these 2 are values of select options :3 i dont know why :D

Reply:
maybe add them as consts on top of file and name them properly, so future user knows, what it does